packages/foam-vscode/jest.config.js

```diff
@@ -82,7 +82,7 @@ module.exports = {
   // moduleNameMapper: {},
 
   // An array of regexp pattern strings, matched against all module paths before considered 'visible' to the module loader
-  // modulePathIgnorePatterns: [],
+  modulePathIgnorePatterns: ['.vscode-test'],
```

Comment:
Otherwise vscode-jest will try to resolve some imports there and fail because of ambiguity (e.g. the yaml module is in every vscode installation).